src/features/bulk/bulk/slice.ts

```diff
@@ -0,0 +1,72 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
I 'm not a fan of moving this away from a `store/` directory. I believe this has been a convention for many years and is instantly recognizable to developers as the place where you go for you redux stuff.

Reply:
I also found examples in react/redux documentation where the slices per feature are found in the feature folder.

Reply:
made a hybrid solution. store folder in the feature.